steps/rhelah.py

```diff
@@ -376,3 +399,69 @@ def step_impl(context, mountpoint):
     '''check whether atomic mount point does not exist'''
     filter_result = find_mount_point(context, mountpoint)
     assert filter_result is not None, "Error unmounted container still exists"
+
+
+@when(u'atomic pull latest "{image}" from repository')
```

Comment:
Since the command that is being used is `atomic update`, I would suggest re-wording this step to say:

```
atomic update latest "{image}" from repository
```